packages/pwa/app/pages/home/index.jsx

```diff
@@ -131,6 +131,11 @@ Home.propTypes = {
     isLoading: PropTypes.bool
 }
 
-Home.getProps = async () => {}
+Home.getProps = async ({res}) => {
+    // Set the `cache-control` header values to align with the Commerce API settings.
```

Comment:
The reason why this comment was made on the product-list page was because we are getting data from the commerce api and we are aligning the cache header value with those that are set in the API. 

On the homepage we aren't using API data at all (except for Einstein data, but that is client side only), so that comment probably doesn't make sense here and we can also use a higher max-age value since the content is static (maybe a year). You should make sure you comment aligns with the value you are setting. 

Since cache is invalidated each time we deploy a bundle, and this content is static, it's safe to make the max age really high, like a year or something. But make sure you reflect the comment text with the reasoning for the value you set on the header.